src/main/java/com/eternalcode/combat/CombatCommand.java

```diff
@@ -25,101 +30,168 @@ public class CombatCommand {
     private final FightBossBarService bossBarService;
     private final NotificationAnnouncer announcer;
     private final PluginConfig config;
+    private final FightTagOutService fightTagOutService;
 
-    public CombatCommand(FightManager fightManager, ConfigService configService, FightBossBarService bossBarService, NotificationAnnouncer announcer, PluginConfig config) {
+    public CombatCommand(FightManager fightManager, ConfigService configService, FightBossBarService bossBarService, NotificationAnnouncer announcer, PluginConfig config, FightTagOutService tagOutService) {
```

Comment:
Dodawanie nowych feature do tej klasy jest mało przyszłościowe. Ten konstruktor jest już i tak giga duży. Proponuję przenieść tagout do osobnego pliku. Litecommands na luzie to złączy.

Reply:
sorry i dont understand polish

Reply:
Lepiej było by znak zapytania napisać :lennyface:

Reply:
?

Reply:
Move the tagout command to separate file

Reply:
ahh okay!
Is it because CombatCommand contructor is too long? Will LiteCommands handle that with ease?